packages/prisma/schema.prisma

```diff
@@ -100,6 +100,7 @@ model Credential {
   // How to make it a required column?
   appId                String?
   destinationCalendars DestinationCalendar[]
+  invalid              Boolean?
```

Comment:
Can we delete the credential if invalid?

Reply:
Yes! Let me elaborate a little more on this: This flag should help us on the backend server when we find out that some credential it's no longer gonna work we can flag it. We can add triggers to this and let know the user to make an action on this.